Makefile

```diff
@@ -65,7 +65,7 @@ BUILD_TAGS := $(wordlist 2,$(words $(BUILD_TAGS)),$(BUILD_TAGS))
 ##                              PROJECT DETAIL                                ##
 ################################################################################
 
-GO_LIST_IMPORT_PATHS_INFO_CMD := go list -tags '$(GO_TAGS)' -f '{{with $$ip:=.}}{{if $$ip.ImportPath | le "github.com/emccode/libstorage"}}{{if $$ip.ImportPath | gt "github.com/emccode/libstorage/vendor" }}{{printf "%s;%s;%s;%s;%v;0,%s;0,%s;0,%s;0,%s" $$ip.ImportPath $$ip.Name $$ip.Dir $$ip.Target $$ip.Stale (join $$ip.GoFiles ",") (join $$ip.TestGoFiles ",") (join $$ip.Imports ",") (join $$ip.TestImports ",")}};{{end}}{{end}}{{end}}' ./...
+GO_LIST_IMPORT_PATHS_INFO_CMD := go list -f '{{with $$ip:=.}}{{if $$ip.ImportPath | le "$(ROOT_IMPORT_PATH)"}}{{if $$ip.ImportPath | gt "$(ROOT_IMPORT_PATH)/vendor" }}{{printf "%s;%s;%s;%s;%v;0,%s,%s,%s,%s;0,%s;0,%s;0,%s" $$ip.ImportPath $$ip.Name $$ip.Dir $$ip.Target $$ip.Stale (join $$ip.GoFiles ",") (join $$ip.CgoFiles ",") (join $$ip.CFiles ",") (join $$ip.HFiles ",") (join $$ip.TestGoFiles ",") (join $$ip.Imports ",") (join $$ip.TestImports ",")}};{{end}}{{end}}{{end}}' ./...
```

Comment:
Hi @clintonskitson,

It was this line that fixed the issue basically. Notice in the old line, it uses the import path `github.com/emccode/libstorage`? Well, when libStorage is built from a vendored location, the import path is everything after `$GOPATH/src` up to, and including, `github.com/emccode/libstorage`. For REX-Ray this would be `github.com/emccode/rexray/vendor/github.com/emccode/libstorage`.